addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5638,7 +5637,7 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
      * @see this.selectClass for parameters
      */
     moveSnippet: function (previewMode, widgetValue, params) {
-        const isMobile = weUtils.isMobileView(this.$target[0]);
+        const isMobile = this._isMobile();
```

Comment:
Note: not sure this is the right approach. Instead of preventing mass_mailing from applying orders when you're editing with a shrunk window, this limits orders to website. That's effectively the same, but that means the whole code is done in web_editor (because column_layout, grid, etc. were defined there) but only has an effect with website. This feels a bit weird.

Reply:
A bit weird indeed but not too concerning, we'll improve mater. Although it's true I forgot about testing mass_mailing with this original work, shame on me :scream: So you confirm that we do not apply order classes over there? :pray: It's right since it surely does not work in all mail clients :pray:

Reply:
Yes, I do confirm!
Before this commit in mass mailing: the order classes were set if working on a small window.
After this commit: they're never added in mass mailing.